lib/puppet/type/group.rb

```diff
@@ -154,17 +154,15 @@ def is_to_s(currentvalue)
       desc "Specify group AIX attributes, as an array of `'key=value'` strings. This
         parameter's behavior can be configured with `attribute_membership`."
 
+      self.log_only_changed_or_new_keys = true
```

Comment:
Given most other type/provider/parameter things are done in a more meta-programmed style, is it possible to define this in a way that this line could be `log_only_changed_or_new_keys true`

Reply:
Oh yeah, that was a mistake on my part. Thanks for pointing it out.

Reply:
Unfortunately, looks like I can't do that without failing rubocop (on a `warning, unused variable assignment` message). So will need to include the self here.

Reply:
Ah, that did not work b/c `attr_accessor` creates a `log_only_changed_or_new_keys` getter method.